openfisca_us/tests/policy/baseline/ssa/ssi/eligibility/income/deemed/from_ineligible_spouse.yaml

```diff
@@ -0,0 +1,56 @@
+- name: Example 1 from https://www.law.cornell.edu/cfr/text/20/416.1163
```

Comment:
Should these be integration tests instead, since they specify other outputs?

Reply:
Hmm, potentially yes. But I think it might be clearer to leave here given they're all from the same source, even though some are technically integration and not unit tests?